ansible/roles/ceph-grafana/defaults/main.yml

```diff
@@ -21,6 +21,11 @@ carbon:
   storage_dir:
     yum: /var/lib/carbon
     apt: /var/lib/graphite
+whisper:
+  retention:
+    - ['10s', '7d']
```

Comment:
Agree. I think the PR should probably include an update to the README so an admin who's installing  picks up on the option to change retention and the implications.